### ISSUE TITLE
Openshift mock server does not save project statuses anymore

### ISSUE THREAD

**Issue by Lutonite** [fabric8io/kubernetes-client#3250]
Hello,

I've recently tried to upgrade to 5.4.1 because of the recent Java TLS changes.

But I'm struggling with the upgrade because the statuses are not stored anymore on the object when I create it.

I have a method which gets an OpenShift project by name and then ensures the status phase is "Active".
So in my UT I created the following:
```java
var status = new ProjectStatus();
status.setPhase("Active");
var project = new ProjectBuilder()
        .withStatus(status)
        .withNewMetadata()
        .withName(ns1cnc.getName())
        .addToAnnotations(NS_ANNOTATION_REQUESTER, "abc")
        .endMetadata()
        .build();
cncClient.projects().create(project);
```

Pre-5.4.0 this wouldn't be an issue, but now the project's status is always null when getting it on the mock server.

This is really blocking for me, I would expect a mock server to be quite dumb and store the status I give it, not remove it somewhere without giving me the option to set it back...

**Comment by shawkins** [comment]
Duplicate of #3144

**Comment by Lutonite** [comment]
@shawkins 

Then when can I expect a release containing the fix? This has blocked my project's upgrade for a while now.

**Comment by shawkins** [comment]
The current fix is targeted to 5.5 which I believe is due in about a month.

**Comment by rohanKanojia** [comment]
Releasing 5.5.0 is already in our sprint plan[0], it should be before 7th July(sprint end)

[0] https://github.com/eclipse/jkube/projects/22
